### PR TITLE
Issue #140: Reduce potential for ambiguity in setMediaKeys definition.

### PR DESCRIPTION
**Pull request by cpearce**
One way to interpret the current setMediaKeys method definition is
if the method is passed a non-null MediaKeys that it should assume
the MediaKeys is in use by another HTMLMediaElement. We can tweak
the wording to make it clear that's not the case.

### CODE REVIEW

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -2067,7 +2067,7 @@
             <li><p>Let <var>promise</var> be a new promise.</p></li>
             <li><p>Run the following steps in parallel:</p>
               <ol>
-                <li><p>If <var>mediaKeys</var> is not null, it is already in use by another media element, and the user agent is unable to use it with this element, let this object's <var>attaching media keys</var> value be false and reject <var>promise</var> with <a def-id="new-domexception-named"></a> <a def-id="QuotaExceededError"></a>.</p></li>
+                <li><p>If <var>mediaKeys</var> is not null, the CDM instance represented by mediaKeys it is already in use by another media element, and the user agent is unable to use it with this element, let this object's <var>attaching media keys</var> value be false and reject <var>promise</var> with <a def-id="new-domexception-named"></a> <a def-id="QuotaExceededError"></a>.</p></li>
```
Comment (ddorwin) [inline encrypted-media-respec.html]: `<var>mediaKeys</var>`

Comment (ddorwin) [inline encrypted-media-respec.html]: Remove "it ".

**Comment by ddorwin** [comment]
Thanks. Two minor comments then LG. Please also update the description to "Fix #..." so the issue will automatically be closed when merged.

**Comment by cpearce** [comment]
Oops, I've pushed a commit to address those comments.

**Comment by ddorwin** [comment]
Merged as d2cccea.

**Pull request closed without merging**